Source/Urho3D/UI/Sprite.h

```diff
@@ -72,6 +72,8 @@ class URHO3D_API Sprite : public UIElement
     void SetRotation(float angle);
     /// Set texture.
     void SetTexture(Texture* texture);
+    /// Set animation.
+    void SetAnimation(const Vector<Pair<SharedPtr<Texture>, unsigned int>> &textures);

```

Comment:
Maybe clarify the comment here about what the first parameter is. Seems like a set of frames, where the second part of the pair indicates approximately how long (in milliseconds?) the frame should be active before the next frame can appear.